deploy-agent/deployd/common/stats.py

```diff
@@ -205,14 +205,17 @@ def serialize(self):
 
     def _deserialize(self):
         """ read in json, setting defaults for a stat
-            return: None
+            return: bool
         """
         obj = json.loads(self.ins)
```

Comment:
better to be explicit here, pass `ins` in the function signature, since this function is internal you can pass self.ins whenever you invoke it. this will give the reader an easier time in the future

Reply:
I agree, I actually visited this idea but it's a lot of refactoring.  I'll make a note to revisit.